libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1213,9 +1273,9 @@ void IRGeneratorForStatements::endVisit(Identifier const& _identifier)
 		else
 			solAssert(false, "Invalid variable kind.");
 	}
-	else if (auto contract = dynamic_cast<ContractDefinition const*>(declaration))
+	else if (dynamic_cast<ContractDefinition const*>(declaration))
 	{
-		solUnimplementedAssert(!contract->isLibrary(), "Libraries not yet supported.");
+		// no-op
 	}
```

Comment:
I guess I will have to add `linkerSymbol()` here when I implement external function calls (as stated in the issue) but in this PR it wasn't yet necessary.